### ISSUE TITLE
Does anyone reproduce the calibration situation of NLL?

### ISSUE THREAD

**Issue by Es7ven** [wkml/MLCC-DCLR#2]
What unit does the calibration indicators presented by the author use? Is it % or ‰? @whwangovo

**Comment by Es7ven** [comment]
Additionally, was the result of NLL obtained by using BCE? @whwangovo

**Comment by Es7ven** [comment]
Loss Composition in the Main Training Script

In the primary training script, the total loss is composed differently based on the configured method:

For the 'MMCE'method:​ The total loss is the sum of a base loss and a calibration loss.

loss_base: Binary Cross-Entropy loss (BCEWithLogitsLoss).

loss_calibration: The MMCE calibration loss.

Total Loss:​ loss = loss_base + loss_calibration

For NLL:​ Only the base loss (BCEWithLogitsLoss) is used. The calibration loss is set to zero.

<img width="694" height="484" alt="Image" src="https://github.com/user-attachments/assets/b194e11d-7700-41ea-ada6-9ed2f5494097" />

**Comment by Es7ven** [comment]
Loss Selection Logic in Calibration.py

In the Calibration.pyfile, the ClassificationAndMDCAclass selects a classification loss based on the lossstring parameter.

If the lossparameter contains "NLL", it uses nn.CrossEntropyLoss().

This selected classification loss is then combined with the MDCAcalibration loss.

<img width="680" height="316" alt="Image" src="https://github.com/user-attachments/assets/e7b93092-4083-4732-99f0-e783483fa882" />

**Comment by Es7ven** [comment]
So, should NLL use BCE or CE? @whwangovo
